src/app.ts

```diff
@@ -13,14 +13,35 @@ interface TemperatureSummary {
   average: number
 }
 
+const data: TemperatureReading[] = []
+
 export function processReadings(readings: TemperatureReading[]): void {
-  // add here your code
+  readings.forEach((item) => {
+    data.push(item)
+  })
 }
 
 export function getTemperatureSummary(
   date: Date,
   city: string,
 ): TemperatureSummary | null {
-  //add here your code
+  const temperatures = data
```

Comment:
I guess it would return null due to the validation with the city property in the temperatureReadings array. But I will add a new local test just to find out what happen.